articles/libraries/standard/error-correction.md

```diff
@@ -129,4 +129,4 @@ using (scratch = Qubit[nScratch]) {
 
 We explore this in more detail in the [bit flip code sample](https://github.com/Microsoft/Quantum/tree/master/Samples/src/BitFlipCode).
 
-Aside from the bit-flip code, the Q# canon is provided with implementations of the [five-qubit perfect code](https://arxiv.org/abs/1305.08), and the [seven-qubit code](https://arxiv.org/abs/quant-ph/9705052), both of which can correct an arbitrary single-qubit error.
+Aside from the bit-flip code, the Q# canon is provided with implementations of the [five-qubit perfect code](https://arxiv.org/abs/1305.0810), and the [seven-qubit code](https://arxiv.org/abs/quant-ph/9705052), both of which can correct an arbitrary single-qubit error.
```

Comment:
It looks like that's a link to one of @vadym-kl's papers on circuit optimization? I think the earliest link for the five-qubit perfect code is https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.77.198, but that's unfortunately paywalled... let me see if I can find a good introductory reference.

Reply:
Yes, this is the paper linked in documentation comments for that code. Do you think that is not the link we want to have in this article?

Reply:
Ah, OK, does sound like the documentation comments are in error, then. Not sure how that happened, sorry.

Reply:
Documentation comments specify where the circuit implemented was taken from, and I've checked that it is the right reference, so that one is fine. We might want to use one of the links in the other documentation: https://docs.microsoft.com/en-us/qsharp/api/qsharp/microsoft.quantum.errorcorrection.fivequbitcode ?